solutions/cses-1076.mdx

```diff
@@ -87,3 +87,28 @@ int main() {
 	cout << endl;
 }
 ```
+
+## Solution 2
+(Arpan Banerjee)
+
+We can directly use an [order statistic tree ](https://codeforces.com/blog/entry/11080) (C++) to get the median while sliding the window across the array.
+```cpp
+#include<bits/stdc++.h>
```

Comment:
This code needs to be formatted (spaces around operators, tabs, common sense stuff).